sceptre/stack.py

```diff
@@ -171,6 +175,14 @@ def update(self):
         :returns: The stack's status.
         :rtype: sceptre.stack_status.StackStatus
         """
+        if self.template.requires_change_set:
+            try:
+                self.get_status()
+            except StackDoesNotExistError:
+                return "PENDING"
```

Comment:
I think that there's a test missing to assert the `PENDING` state (it also looks like this may not really be used, still good to have)